Formula/gq-cli.rb

```diff
@@ -0,0 +1,33 @@
+class GqCli < Formula
+  desc "Command-line interface for GQ"
+  homepage "https://goquant.io"
+  license "MIT"
+
+  bottle do
+    sha256 cellar: :any_skip_relocation, monterey: "b21e0a5f4238130218bfae10c6604a05263e9ef52f462af246d54a93f54821d1"
+  end
+
+  on_macos do
+    on_arm do
+      url "https://brickelldatabros1aboah.blob.core.windows.net/public/gq-arm-0.0.1.big_sur.bottle.tar.gz"
+      sha256 "d8d14a758174c9f67bb7adaf4d4a8897f3b3fa3b7f4338eb86f2c2c222e9a785"
+    end
+
+    on_intel do
+      url "https://brickelldatabros1aboah.blob.core.windows.net/public/gq-x86-0.0.1.high_sierra.bottle.tar.gz"
+      sha256 "99a64a8b7164c66a03aa8f34e68a7acb4cf2254642345839c742afd989a5f2f1"
+    end
+  end
```

Comment:
yes this software also supports Linux. If the formula is installed then it should natively compile to the required architecture.
I would like to know more about serving Bottles (binaries) as building from source can be strenuous on older hardware.

Reply:
Bottles are pre-built binary packages that we build on CI (you can see the CI status at the end of this PR). If a user installed Homebrew at the default, supported location, they don't need to build the software from source.

> When the formula being installed defines a bottle matching your system, it will be downloaded and installed automatically when you run `brew install <formula>`.

> Bottles for `homebrew/core` formulae are created by [Brew Test Bot](https://docs.brew.sh/Brew-Test-Bot) when a pull request is submitted. If the formula builds successfully on each supported platform and a maintainer approves the change, Brew Test Bot updates its `bottle do` block and uploads each bottle to [GitHub Packages](https://github.com/orgs/Homebrew/packages).

Reference: https://docs.brew.sh/Bottles.